In [12]:
from scipy.stats import norm
import numpy as np

<hr style="color: #009933; border: solid 1px">
<span style="color: #009933;">known std, $\sigma$</span>

# <span style="color: #2455C3">Valentine's Day</span>

A flower shop noted that its average sales every Valentine's day are \$7,895 with standard deviation \$230. The last 5 years, they have implemented a new marketing strategy to hopefully increase sales. Their average sales over the las 5 years is \$9,640.

$$\begin{array}{cl}
H_0: & \mu = \mu_I \\
H_1: & \mu \neq \mu_I
\end{array}$$

## <span style="color: #85100F">Solution</span>

#### <span style="color: #85100F">1. At $\alpha = 0.01$, what is the probability of obtaining at least this mean?</span>

In [3]:
# known data
# one sample two-tailed test
n = 5.0
mu = 7895.0
sigma = 230.0
x_bar = 9640.0
alpha = 0.01

In [4]:
SE = sigma / np.sqrt(n)
SE

102.85912696499032

In [5]:
# test statistic calculation
z_score = (x_bar - mu) / SE
z_score

16.964950524944058

In [6]:
# critical point determination
z_critical = norm.ppf(1-alpha/2.0)
z_critical

2.5758293035489004

#### <span style="color: #85100F">2. What is the statistical decision?</span>

<span style="color: #009933;">The <b>null hypothesis is rejected with $p<0.001$</b>, which means that <b>the marketing strategy had an effect on Valentine's day sales, increasing it</b>.</span>

1) <u>Descriptive statistics:</u>
    $$\begin{array}{ccc}
    \mu & = & \$7,895.00\\
    \bar{x} & = & \$9,640.00\\
    \sigma & = & \$230.00
    \end{array}$$

2) <u>Inferential statistics:</u>

z=16.96, p=.00, two-tailed<br>
Confidence interval on the mean sales of Valentine's day<br>
95% CI = ($9,375.05 - $9,904.95)

3) <u>Effect size measures:</u>

* Cohen's d = 7.59

In [11]:
# P-value
P_value = 2*norm.sf(z_score)
print "P-value = {}".format(P_value)

P-value = 1.49225308601e-64


In [9]:
# CI
z_char = norm.ppf(1-alpha/2)
me = z_char * SE
print "95% CI = ({:0.2f} - {:0.2f})".format(x_bar-me, x_bar+me)

95% CI = (9375.05 - 9904.95)


In [10]:
# Cohen's d
d = (x_bar - mu) / sigma
print "d = {:0.2f}".format(d)

d = 7.59


<hr style="color: #009933; border: solid 1px">
<span style="color: #009933;">unknown std, $s$</span>